app/cdap/components/WranglerV2/RecipeStepsTable/RecipeStepsTable.stories.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { action } from '@storybook/addon-actions';
+import RecipeStepsTable, {
+  IRecipeStepsColumns,
+  IRecipeStepsRows,
+} from 'components/WranglerV2/RecipeStepsTable';
+import React from 'react';
+
+export default {
+  title: 'RecipeStepsTable',
+  component: RecipeStepsTable,
+};
+
+// Recipe Steps Panel with Recipe Steps Table
+const handleDeleteIconClick = (row: IRecipeStepsRows) =>
+  action('clicked')(`Delete Icon Button Clicked with id = ${row.id}`);
+
+const dataGridColumns: IRecipeStepsColumns[] = [
+  {
+    field: 'srn',
+    headerName: '#',
+    sortable: false,
+    width: 71,
+  },
+  {
+    field: 'step',
+    headerName: 'Recipe Steps',
+    sortable: false,
+    width: 389,
+  },
+];
+
+const dataGridRows: IRecipeStepsRows[] = [
+  {
+    id: 1,
+    step: "Parse Column 'Body' with delimiter 'comma' and set 'first row as header' ",
+    srn: '01',
+  },
+  { id: 2, step: "Delete Column 'body'", srn: '02' },
+];
+
+export function RecipeStepsPanel(args) {
+  return <RecipeStepsTable {...args} />;
+}
+
+RecipeStepsPanel.args = {
```

Comment:
Where did you see the example of doing it like this?

Reply:
@seanfeld 

We are following the example given in the official documentation of the storybook,
Here is the link  - [https://storybook.js.org/docs/react/get-started/whats-a-story]

Reply:
It looks like you're not following the typescript examples

Reply:
updated the code according to the typescript examples